### ISSUE TITLE
Assertion `(x >= static_cast<T>(0)) && (x <= one)` failed. Input is expected to be within the interval [0, 1], but recieved nan.

### ISSUE THREAD

**Issue by Jeremy-lf** [PaddlePaddle/PaddleDetection#6464]
[07/15 22:35:03] ppdet.engine INFO: Epoch: [3] [120/922] learning_rate: 0.000404 angle_loss: 5.099063 phi_cos_loss: 1.231465 phi_sin_loss: 1.642504 occluded_loss: 7.705573 truncated_loss: 6.232909 occluded_acc: 0.610000 truncated_acc: 0.494687 heatmap_loss_mean: 34.394241 yolo_loss: 114.108925 loss: 172.359558 eta: 1 day, 18:09:49 batch_cost: 13.3108 data_cost: 7.6279 ips: 1.2020 images/s
-------
-------
-------
-------
-------
-------
-------
-------
-------
-------
-------
Error: /paddle/paddle/phi/kernels/gpu/bce_loss_kernel.cu:42 Assertion `(x >= static_cast<T>(0)) && (x <= one)` failed. Input is expected to be within the interval [0, 1], but recieved nan.
Error: /paddle/paddle/phi/kernels/gpu/bce_loss_kernel.cu:42 Assertion `(x >= static_cast<T>(0)) && (x <= one)` failed. Input is expected to be within the interval [0, 1], but recieved nan.
Error: /paddle/paddle/phi/kernels/gpu/bce_loss_kernel.cu:42 Assertion `(x >= static_cast<T>(0)) && (x <= one)` failed. Input is expected to be within the interval [0, 1], but recieved nan.
Error: /paddle/paddle/phi/kernels/gpu/bce_loss_kernel.cu:42 Assertion `(x >= static_cast<T>(0)) && (x <= one)` failed. Input is expected to be within the interval [0, 1], but recieved nan.
Error: /paddle/paddle/phi/kernels/gpu/bce_loss_kernel.cu:42 Assertion `(x >= static_cast<T>(0)) && (x <= one)` failed. Input is expected to be within the interval [0, 1], but recieved nan.

![image](https://user-images.githubusercontent.com/45171399/179439245-15d792ac-0cbe-42e5-a57b-2c2e38b25c4d.png)

paddledetection，训练yolov3框架检测算法，训练了3个epcho后报错，什么问题呢？
如流：lvfeng02

### 是否愿意提交PR Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by lyuwenyu** [comment]
跑飞了吧，  提供 paddle版本 ppdet版本  以及使用的是什么数据集，

**Comment by Jeremy-lf** [comment]
> 
paddle2.3 ，ppdet2.3，任务是一阶段检测+二阶段属性预测

**Comment by lyuwenyu** [comment]
把LR调小再试一下吧

**Comment by lemon9707** [comment]
> paddle2.3 ，ppdet2.3，任务是一阶段检测+二阶段属性预测
请问解决了吗，我同样的问题

**Comment by lemon9707** [comment]
> paddle2.3 ，ppdet2.3，任务是一阶段检测+二阶段属性预测
请问解决了吗，我同样的问题

**Comment by lemon9707** [comment]
@Jeremy-lf
